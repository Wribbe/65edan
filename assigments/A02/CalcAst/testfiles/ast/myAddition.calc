let a=2.0 b=4.0 c=20.0 in 
    a * b * c * 10.0
end
