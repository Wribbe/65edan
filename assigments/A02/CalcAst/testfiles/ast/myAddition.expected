Program
  Let
    List
      Binding
        IdDecl ID="a"
        Numeral NUMERAL="2.0"
      Binding
        IdDecl ID="b"
        Numeral NUMERAL="4.0"
      Binding
        IdDecl ID="c"
        Numeral NUMERAL="20.0"
    Mul
      Mul
        Mul
          IdUse ID="a"
          IdUse ID="b"
        IdUse ID="c"
      Numeral NUMERAL="10.0"
